disnake/channel.py

```diff
@@ -2285,6 +2285,7 @@ async def create_thread(
                 name=name,
                 auto_archive_duration=auto_archive_duration or self.default_auto_archive_duration,
                 rate_limit_per_user=slowmode_delay or 0,
+                type=ChannelType.public_thread.value,
```

Comment:
why was this added? type cannot be private in a forum channel.

Reply:
okay, I understand now, and this makes sense. However, I'm not sure we should be specifying it, but I also don't believe it matters.